builtin/logical/pki/cert_util.go

```diff
@@ -432,31 +444,84 @@ func validateNames(req *logical.Request, names []string, role *roleEntry) string
 	return ""
 }
 
+func validateOtherSANs(data *dataBundle, requested map[string][]string) (string, string, error) {
```

Comment:
Can we have a comment for this method mentioning that this will performing the regex check on the allowed names on the role?

Reply:
Additionally it wasn't clear, until looking at the caller, what the return values represent. Could you document that?